### DISCUSSION TITLE
SRS gNB Setup Request Rejected by Flexric

### DISCUSSION THREAD

**Discussion by LVancina** [srsran/srsRAN_Project#326] [General Help]
Greetings,

I am new to srsRAN and am working through one of the example experiments from the documentation (https://docs.srsran.com/projects/project/en/latest/tutorials/source/flexric/source/index.html). I have everything installed and I was able to run the 5gc and Flexric. However, when I attempt to run the gNB, I get the following output on the Flexric terminal:

Received message with id = 411, port = 22429
[E2AP] Received SETUP-REQUEST from PLMN   1. 1 Node ID 411 RAN type ngran_gNB
nearRT-RIC: /users/lvancina/flexric/src/ric/msg_handler_ric.c:88: generate_setup_response: Assertion `req->ran_func_item[i].def.len < 127' failed.
Aborted

Any ideas why this might be happening?

I will add that I am running the gnb executable from the top level srsRAN directory (where I downloaded the gnb_zmq.yaml file, using the command sudo ./build/apps/gnb/gnb -c gnb_zmq.yaml. I will not be surprised if this is wrong, but the instructions weren't very clear about where to put the gNB config file or how to run the gNB. Any pointers would be appreciated.

Thanks!

**Comment by pgawlowicz** [comment]
Hi, please make sure you have applied the FlexRIC patch provided in the app note.
